test/e2e/app-dir/next-after-app-invalid-usage/index.test.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jest/no-standalone-expect */
+/* eslint-env jest */
+import { nextTestSetup } from 'e2e-utils'
+import * as fs from 'fs'
+import * as path from 'path'
+import * as os from 'os'
+import * as Log from './utils/log'
+import {
+  assertHasRedbox,
+  getRedboxDescription,
+  getRedboxSource,
+} from '../../../lib/next-test-utils'
+
+describe('unstable_after() - invalid usages', () => {
+  const logFileDir = fs.mkdtempSync(path.join(os.tmpdir(), 'logs-'))
+  const logFile = path.join(logFileDir, 'logs.jsonl')
+
+  const { next, isNextDev, skipped } = nextTestSetup({
+    files: __dirname,
+    // `patchFile` and reading runtime logs are not supported in a deployed environment
+    skipDeployment: true,
```

Comment:
`patchFile` is not used anymore. but maybe this whole test could be moved to `test/development` because the tests are skipped anyways if `isNextDev` is false? although I do wonder why we're not testing the `build` errors for these two scenarios. 🤔

Reply:
honestly, i mostly skipped `build` because it's harder to test. the build bails out on the `"use client"` page and doesn't even try to do the other ones. so i'd probably need to split it into three apps... don't think we're losing much here though. i guess moving it to `test/development` is fine